src/Interfaces/Chakra__Theme.res

```diff
@@ -79,4 +79,12 @@ module Size = {
       | xs
     ]
   }
+  module Select = {
+    type t = [
+      | xs
+      | sm
+      | md
+      | lg
```

Comment:
As per chakra docs, only 4 sizes for a select.

Reply:
Thankyou @tjbo